libraries/kotlin-spark.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "scala": "2.12.10",
+    "spark": "3.0.0",
+    "kotin-spark": "0.3.0"
+  },
+  "dependencies": [
+    "org.apache.spark:spark-repl_2.12:$spark",
```

Comment:
It seems to me that scala version (2.10, 2.11, 2.12) should be a property, and the same version should be used for all dependencies

Reply:
Currently, only 2.12 is supported, so I think we should leave it as is at the time being.